--- inst_dirinstall_finish.ycp.orig     2013-01-22 11:17:27.000000000 +0100
+++ inst_dirinstall_finish.ycp  2014-04-20 15:54:41.529209645 +0200
@@ -17,7 +17,6 @@
     import "Installation";
     import "Directory";
     import "Progress";
-    import "Mouse";
     import "Keyboard";
     import "Language";
     import "Timezone";
@@ -85,7 +84,6 @@
         y2error ("ldconfig failed\n");

     SCR::Execute (.target.mkdir, Installation::sourcedir);
-    Mouse::Save();
     Timezone::Save();
     Language::Save();
     Keyboard::Save();
